packages/indicators/src/__tests__/__integration/arithmeticAnalytics.fixtures.json

```diff
@@ -99,6 +95,64 @@
             "formula": "Male > Female",
             "aggregation": "FINAL_EACH_MONTH"
           }
+        },
+        {
+          "code": "TotalBirths",
+          "builder": "arithmetic",
+          "config": {
+            "formula": "MonthlyBirths",
+            "aggregation": "SUM"
+          }
+        },
+        {
+          "code": "QuarterlyIncrease",
+          "builder": "arithmetic",
+          "config": {
+            "formula": "TotalBirths - prevQuarterTotalBirths",
+            "parameters": [
+              {
+                "code": "prevQuarterTotalBirths",
+                "builder": "arithmetic",
+                "config": {
+                  "formula": "TotalBirths",
+                  "aggregation": [
+                    { "type": "OFFSET_PERIOD", "config": { "offset": 1, "periodType": "QUARTER" } }
+                  ]
+                }
+              }
+            ],
+            "aggregation": "FINAL_EACH_QUARTER"
+          }
+        },
+        {
+          "code": "AverageOf3QuarterlyIncreases",
```

Comment:
This is a cool demonstration of the power of all of the pieces!

Reply:
Indeed! It exposes a lot of aspects of nested aggregations